docs/qa/CheckLists/FavoritePage_tags.md

```diff
@@ -0,0 +1,68 @@
+| №  | Description                                                                                                                                                                                                                                              | Requirement | Commentary                                                                         |
+| -- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- | ---------------------------------------------------------------------------------- |
+|    | 1.1 Favorite empty opening page 11                                                                                                                                                                                                                       |             |                                                                                    |
+| 1  | Text "here will be your added words as soon as you create tags and start adding favorites" is presented                                                                                                                                                  |             |                                                                                    |
```

Comment:
It's better to give a short and clear name for the hint instead of using its entire text